### ISSUE TITLE
Alarm - power outage

### ISSUE THREAD

**Issue by hauhan** [alexa/avs-device-sdk#1399]
## Briefly summarize your issue:
According to test item 239 (Alarm - power outage)
Set alarm or timer then power off device before time over, when the alarm time pass then power on device that AVS should be alarm.

## What is the expected behavior?
Alarm should be triggered after connect to Alexa.

## What behavior are you observing?
No any alarm or timer will be triggered.
(About one year ago, this test was pass)

## Provide the steps to reproduce the issue, if applicable:
(Current Time 04:46 AM)
Ask Alexa, set alarm in 04:50 AM
(Ask Alexa what's my alarm, then Alexa say your alarm is 04:50 AM)
power OFF
(Current Time 05:01 AM)
power ON
Alexa is connected but no alarm be triggered.

### What version of the AVS Device SDK are you using?
AVS SDK 1.6 (yes, it is old)

### Tell us what hardware you're using:
ARM

### Tell us about your OS (Type & version):
embedded linux

**Comment by scotthea-amazon** [comment]
Hello Hauhan,

I have build SampleApp from 1.6 source and after several tries (using kill to simulate a power outage) I have not been unable to reproduce this problem.  If you provide DEBUG9 logs for a reproduction of the problem we might be able to get a better idea of what is happening.

Of course, I also have to recommend upgrading to a more recent release of the SDK.  There have been many bug fixes in the intervening releases.

Thank you,
-SWH

**Comment by hauhan** [comment]
Hello Scotthea,

We fixed this bug already, because we changed .db files (alerts.db, notification.db etc.)position from flash to memory that every time reboot will recreate .db files.

start alexa service
Running app with log level: DEBUG9
2019-06-26 01:46:20.952 [  1] 0 ConfigurationNode:initializeSuccess
2019-06-26 01:46:21.272 [  1] E SQLiteDatabase:open::File specified does not exist.:file path=/tmp/certifiedSender.db
2019-06-26 01:46:21.272 [  1] I CertifiedSender:init : Database file does not exist.  Creating.
2019-06-26 01:46:21.293 [  1] E SQLiteDatabase:open::File specified does not exist.:file path=/tmp/alerts.db
2019-06-26 01:46:21.294 [  1] I AlertScheduler:initialize::Couldn't open database.  Creating.
2019-06-26 01:46:21.306 [  1] E SQLiteDatabase:open::File specified does not exist.:file path=/tmp/notification.db

Thanks for your help.

Hauhan
